package.json

```diff
@@ -5,7 +5,7 @@
   "license": "(MIT OR Apache-2.0)",
   "scripts": {
     "dev": "next dev",
-    "build": "node scripts/purgableCSSGenerator.js && next build",
+    "build": "yarn lint && node scripts/purgableCSSGenerator.js && next build",
```

Comment:
I believe Next.js runs lint automatically when building the app, so we don't need to run it twice:
https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint

Reply:
oh, didn't know that. Shall we remove this extra step from our PR workflow as well then?